docs/resources/ipam_address_block.md

```diff
@@ -85,6 +91,9 @@ resource "bloxone_ipam_address_block" "example_tags" {
       }
     ]
   }
+
+  //federated realms
```

Comment:
Unnecessary comment.

Same for subnet and ip space.

Reply:
done